packages/hint/docs/contributor-guide/how-to/formatter.md

```diff
@@ -25,17 +25,24 @@ A `message` looks like this:
 
 ```json
 {
-    "column": "number", // The column number where the issue was found if applicable.
-    "line": "number", // The line number where the issue was found if applicable.
+    "location": {
+        "column": "number", // The column number where the issue was found if applicable.
+        "line": "number", // The line number where the issue was found if applicable.
+    },
     "message": "string", // The human friendly detail of the error.
+    "sourceCode": "string", // The html element code where the issue was found if applicable.
```

Comment:
This doesn't need to be the `html element code`.

what about `The piece of code where the issue was found if applicable`

Reply:
Updated